### PR TITLE
HDEV 384 - disable vesting on arb

### PR DESCRIPTION
**Pull request by schystz**
## Description of Changes

- Disable vesting page when network selected is Arbitrum or Arb testnet
- Fix explorer link for Arb & Arb testnet

[Link to Jira Ticket](https://halodao.atlassian.net/browse/HDEV-384)

### Developer Checklist:

* [x] I have followed the guidelines in our Contributing document
* [x] This PR has a corresponding JIRA ticket
* [x] My branch conforms with our naming convention i.e. `feature/HDEV-XXX-description`
* [x] All files have appropriate file headers and documentation
* [ ] I have written new tests for your core changes, as applicable
* [ ] I have successfully ran tests locally

### Reviewers Checklist:
* [ ] Code is readable and understandable; any unclear parts have explanations 
* [ ] UI/UX changes match the corresponding figma/other design resources, if applicable
* [ ] I have successfully ran tests locally

### CODE REVIEW

**Review by tracyarciaga — APPROVED** [review]

File: src/components/Tailwind/Panels/FeatureNotSupported.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import PageWarning from '../Layout/PageWarning'
+
+interface FeatureNotSupportedProps {
+  isIsolated: boolean
+}
+
+const FeatureNotSupported = ({ isIsolated }: FeatureNotSupportedProps) => {
+  const { chainId } = useActiveWeb3React()
+
+  const caption = 'Ooops! Sorry this feature is not supported in ' + NETWORK_LABEL[chainId as ChainId] + '.'
```
Comment (bitcoinbrisbane) [inline src/components/Tailwind/Panels/FeatureNotSupported.tsx]: Should we use a template?

```suggestion
  const caption = `Ooops! Sorry this feature is not supported in ${NETWORK_LABEL[chainId as ChainId]}.`
```

**Review by bitcoinbrisbane — COMMENTED** [review]
One suggestion, and looks like some linting issues to fix.

**Review by mystbrent — APPROVED** [review]
LGTM!

File: src/components/Tailwind/Panels/FeatureNotSupported.tsx (hunk 1 above)
Comment (schystz) [reply src/components/Tailwind/Panels/FeatureNotSupported.tsx]: Yep, better!

**Pull request merged**